nixos/modules/services/security/privacyidea.nix

```diff
@@ -57,6 +57,18 @@ in
     services.privacyidea = {
       enable = mkEnableOption "PrivacyIDEA";
 
+      environmentFile = mkOption {
+        type = types.nullOr types.path;
+        default = null;
+        example = "/root/privacyidea.env";
+        description = ''
+          File to load as environment file. Environment variables
+          from this file will be interpolated into the config file
+          using envsubst with this syntax:
+          <literal>$ENVIRONMENT ''${VARIABLE}</literal>
```

Comment:
This example is confusing: it explains how to set the _other_ options in order to have them substituted in; I'd prefer a fuller explanation (with an example along the lines of `services.privacyidea = {secretKey = "$SECRET_KEY"; environmentFile = "/root/privacyidea.env";}` plus sample contents of `/root/privacyidea.env`), or just leaving it entirely up to the user to work it out (cut off after `… using envsubst.`)